ui/pages/token-allowance/token-allowance.js

```diff
@@ -413,7 +434,13 @@ export default function TokenAllowance({
         onCancel={() => handleReject()}
         onSubmit={() => (isFirstPage ? handleNextClick() : handleApprove())}
         disabled={disableNextButton || disableApproveButton}
-      />
+      >
+        {unapprovedTxCount > 1 && (
+          <a onClick={() => handleCancelAll()}>
```

Comment:
To make this accessible, this element needs a `href` attribute. So maybe something like:

```
<a href="" onClick={(e) => { e.preventDefault(); handleCancelAll() }}>
```

Better yet would be to use a `<Button>` component.